.gitignore

```diff
@@ -58,4 +58,4 @@ docs/_build/
 # PyBuilder
 target/
 
-local_settings.py
+/local_settings.py
```

Comment:
https://git-scm.com/book/en/v2/Git-Basics-Recording-Changes-to-the-Repository#Ignoring-Files

You can start patterns with a forward slash (/) to avoid recursivity.